keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1878,84 +1909,252 @@ progress.
 
 ### Cluster Autoscaler
 
+-<<[UNRESOLVED pohly]>>
```

Comment:
FYI, I want to clarify that, given that this entire section is marked as UNRESOLVED, and sig-autoscaling seems mostly opposed to the idea of compiled-in plugins, I won't be accepting the code changes for kube-scheduler suggested in this section.

You can still iterate on the design during the release.

Reply:
As discussed after KubeCon NA 2023 and in Batch WG, the intended solution for cluster autoscaling is to make parameters less opaque via "numeric parameters". This is, at least from a technical perspective, an extension of the core DRA. Therefore I have created a [separate KEP #4381](https://github.com/kubernetes/enhancements/issues/4381) for it and in this PR pushed an update that makes it clear that cluster autoscaling will not work as expected when using opaque parameters.

If we agree on this approach, then this PR might be ready for merging.

https://github.com/kubernetes/enhancements/pull/4181#issuecomment-1750755469:

> I would like to see an exploration of how those two concepts can be detached.

I've taken this into account during the design of numeric parameters. Let's head over to the [initial PR for numeric parameters](https://github.com/kubernetes/enhancements/pull/4384) and continue there...